src/SnackbarContent/SnackbarContent.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { forwardRef } from 'react';
 import clsx from 'clsx';
-import { styled } from '@material-ui/core/styles';
+import { styled } from '@mui/styles';
```

Comment:
here too... and a few places below

Reply:
Change it to `import { styled } from '@mui/material';`